clone/clone.go

```diff
@@ -25,14 +26,18 @@ type Error struct {
 	Details string
 }
 
+func (e Error) Error() string {
+	return fmt.Sprintf("%s:\n%s", e.error.Error(), e.Details)
+}
+
 func Clone(ctx context.Context, input Input) (Output, error) {
 	cloneIntoDir := path.Join(input.WorkDir, "cloned")
 	if _, err := os.Stat(cloneIntoDir); err == nil {
 		// already cloned
 		return Output{Success: true, ClonedIntoDir: cloneIntoDir}, nil
 	}
 
-	cmd := exec.CommandContext(ctx, "git", "clone", input.GitURL, cloneIntoDir)
+	cmd := exec.CommandContext(ctx, "git", "clone", "--quiet", input.GitURL, cloneIntoDir)
```

Comment:
What's the purpose of the --quiet addition?

Reply:
Ah I thought it was worth having more focused output on error but it does seem like in most error cases it's not too much output and we can let the user look over the full (non quiet) log.

For example the non-quiet output for the whitespace issues is:
```
2025/01/06 10:20:18 exit status 128:
Cloning into '/Users/taylor.sutton/go/src/github.com/Clever/microplane/mp/catapult /clone/cloned'...
fatal: remote error: 
 Clever/catapult  is not a valid repository name
Visit https://support.github.com/ for help
```
and the quiet output just removes one line
```
2025/01/06 10:20:18 exit status 128:
fatal: remote error: 
 Clever/catapult  is not a valid repository name
Visit https://support.github.com/ for help
```

So I'll remove the quiet.